packages/preview/usaf-memo/0.0.1/README.md

```diff
@@ -0,0 +1,200 @@
+# USAF Memo Template for Typst
+
+A comprehensive Typst template for creating official United States Air Force memorandums that comply with AFH 33-337 "The Tongue and Quill" formatting standards.
+
+## Features
+
+- **Automatic formatting compliance** with AFH 33-337 standards
+- **Hierarchical paragraph numbering** (1., a., (1), (a))
+- **Professional typography** with Times New Roman font and proper spacing
+- **Complete letterhead automation** including DoD seal placement
+- **Flexible content management** for various memo types
+- **Proper signature block positioning** and closing elements
+
+## Quick Start
+
+### Installation
+
+```bash
+# Clone the repository
+git clone https://github.com/SnpM/typst-usaf-memo.git
+cd typst-usaf-memo
+
+# Compile the example
+typst compile lib.typ
+```
+
+### Basic Usage
+
+```typst
+#import "lib.typ": *
```

Comment:
You should demonstrate how to use the `@preview` import in your examples, since this will be the easiest way to use your package.